### PR TITLE
Adding extra filter and parameter

### PR DESCRIPTION
**Pull request by alexowson**
Adding keywords filter to instabot.py
Adding start and end time parameters to run.py

### CODE REVIEW

**Comment by maxatbrs** [comment]
@alexowson I took a look and keywords stuff looks pretty straight forward. What is the purpose of the `activate.py` file?

**Comment by alexowson** [comment]
activate.py file is not necessary. I created it for personal purpose.

File: run.py [hunk 1]
```diff
@@ -41,6 +41,11 @@ def setupinteractive(config, config_location="config.ini"):
         "unfollow_break_min": "opt",
         "unfollow_break_max": "opt",
         "tag_list": "opt",
+        "keywords": "opt",
+        "start_at_h": "opt",
+        "start_at_m": "opt",
+        "end_at_h": "opt",
+        "end_at_m": "opt",
```
[the remark this answers is not in the document]
Comment (alexowson) [reply run.py]: Ups! I added extra code in next commits :P

**Comment by alexowson** [comment]
So. I will prepare a branch only with keywords stuff and i will send a new request :)

**Comment by alexowson** [comment]
Done. : #2005

**Pull request closed without merging**